sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_options.hpp

```diff
@@ -401,4 +401,17 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     std::string const& Name() const { return Properties.Name; }
   };
 
+  /**
+   * @brief Get Random Bytes options
+   *
+   */
+  struct GetRandomBytesOptions final
+  {
+    /**
+     * @brief The requested number of random bytes.
+     *
+     */
+    int32_t Count;
```

Comment:
we use uint64_t for parameters that are sent to the network.  Is there a reason why we need int32_t?

Reply:
in the same file at the top there is an int64_t ;)  
.Net also uses System.Int32. The documentation specifies an int32 to be used.